common/js/auth-buttons.js

```diff
@@ -30,7 +30,7 @@
         await auth.logout()
       }
       // Now that we have a cookie, reload to display the authenticated page
-      location.reload()
+      location.href = new URL(session.webId).origin
```

Comment:
If I remember correctly, this ended up being overzealous as you would rather have the same page reload rather than redirecting to the origin.